### DISCUSSION TITLE
Add a "Jobs" forum category?

### DISCUSSION THREAD

**Discussion by yajo** [OCA/odoo-community.org#95] [Polls]
Hi folks!

I've seen in other similar communities a "Jobs" category in their forums, so companies looking for employees that have experience in a specific technology can post their offers directly where these people communicate.

I think it would be helpful and give value to this community. Many people hanging around OCA will probably want to start creating or improving their career by making Odoo their full time job. As a community, I think it's good to open the door for helping in this important part of the life of our contributors.

TBH I've been working with Odoo for the last few years thanks to starting contributing in the OCA. I'm now in position of hiring somebody else and I also think this is a valuable resource of talent. I just want to do it transparently if possible.

However, I've asked other folks and some say that this could be considered a spammy behavior. I don't think it's spam because we're not trying to sell or steal personal data, but I want to know what you all think about the subject.

That's why I wanted to open this poll and thread:
1. Do you see useful adding this "Jobs" category?
2. Would you consider spam publishing job offers in the forum?

Please vote and feel free to post your opinions.

Thanks!

**Comment by ap-wtioit** [comment]
I think it is a good idea to have a jobs category if jobs postings are accepted. I can see why people would be concerned that job postings would increase spam. E.g. most of the contact requests over linkedin regarding Odoo i consider spam. (Not if i have met you in person ;-)) However i would not classify all job postings and replies to those as spam themselves just for being job related. So i think it would be worth talking the risk and see if something positive comes out of it. (With an option to stop it if it's mainly used for spam/scam)

**Reply by Rad0van** [reply]
I think it's a good idea. It's just some measures should be incorporated to get relevant answers to those job posts. Otherwise it will quickly become similar to LinkedIn where you get a lot of spam.

**Reply by yajo** [reply]
Yeah... I wish one could just subscribe to some categories in the forum, and not to others... I guess that'll come some day from the Github devs.

What kind of measures do you think that could help with this?

I also asked about [the same thing in the AEOdoo forum](https://www.aeodoo.org/forum/general-1/publicar-ofertas-de-empleo-451) and the first idea that bubbled up was to indicate how this could be a feature from the association itself. 

It could even be a paid service, or a service offered only to associates of some tier. That by itself would make the feature less attractive to spammers, and could be a way to get revenue by offering a practical service to the OCA members.

The truth is that, as members, most of us need to work or to hire. I think that helping somehow in that process would make OCA more valuable.

**Comment by rvalyi** [comment]
Hello, I would also like to warn about what is called a "market for lemon" in case there is no control of any sort, or in more popular term: prostitute the market to death. This how a market use to die when service suppliers (us) have more information then the average customer.
Please read carefully: https://saylordotorg.github.io/text_introduction-to-economic-analysis/s19-01-market-for-lemons.html

Today the OCA is a civilized place where liars and crooks are not welcome, not so different how was the Odoo partner network say until 10 years ago. Let's be very careful not changing it into the official Odoo partner network or sales or we could easily kill much of its value... So I'm not against it, but only if it is very well bounded.
